firmware/src/macros.h

```diff
@@ -35,4 +35,13 @@
 #define UNUSED
 #endif
 
+/**
+ * @brief A helper to return if the request does not merit a response.
+ * @param header The RDMHeader of the request.
+ */
+#define ReturnUnlessUnicast(header) \
```

Comment:
It was in rdm_responder.c, I moved it here so I can reuse it for all the models.